Phonebook.Frontend/package-lock.json

```diff
@@ -17039,6 +17040,12 @@
             }
           }
         },
+        "marked": {
```

Comment:
Please do not modify or check in changes to the package-lock.json without changing the package.json